param/reactive.py

```diff
@@ -1961,6 +1961,11 @@ def _eval_operation(self, obj, operation):
             obj = fn(obj, *resolved_args, **resolved_kwargs)
         return obj
 
+    def __setattr__(self, name, value):
+        if name == "value":
```

Comment:
Why only value?

And what if the underlying object itself has a value attribute?

Reply:
> And what if the underlying object itself has a value attribute?

I don't think this has any consequence? Currently, for an object that has a `value` attribute, `rx(obj).value` returns a reactive expression. Setting `rx(obj).value` sets `value` on the reactive expression, not on the underlying object, which can be done with `rx(obj).rx.value = <value>`. Let me know if I got this wrong.

> Why only value?

Because that's the most error-prone attribute in my opinion and I don't want to restrict `__setattr__` too much beyond that.

Reply:
> Because that's the most error-prone attribute in my opinion and I don't want to restrict __setattr__ too much beyond that.

A bit opinionated, but I think I agree. Can you add a comment in the code about this?